packages/editor/src/lib/exports/exportToSvg.tsx

```diff
@@ -0,0 +1,67 @@
+import { TLShapeId } from '@tldraw/tlschema'
+import { assert } from '@tldraw/utils'
+import { flushSync } from 'react-dom'
+import { createRoot } from 'react-dom/client'
+import { Editor } from '../editor/Editor'
+import { TLSvgOptions } from '../editor/types/misc-types'
+import { getSvgJsx } from './getSvgJsx'
+import { decorateAndEmbed } from './html-to-image/decorateAndEmbed'
+import { FontEmbedder } from './html-to-image/embedCss'
+
+export async function exportToSvg(editor: Editor, shapeIds: TLShapeId[], opts: TLSvgOptions = {}) {
+	const result = await getSvgJsx(editor, shapeIds, opts)
+	if (!result) return undefined
+
+	const container = editor.getContainer()
+	const renderTarget = document.createElement('div')
+	renderTarget.className = 'tldraw-svg-export'
+	renderTarget.inert = true
+	renderTarget.tabIndex = -1
+	Object.assign(renderTarget.style, {
+		position: 'absolute',
+		top: '0px',
+		left: '0px',
+		width: result.width + 'px',
+		height: result.height + 'px',
+		zIndex: '110000000',
```

Comment:
very specific zIndex :P might as well make it 2147483647?

Reply:
this was just for debugging - removed!